tools/pinning/current/pyproject.toml

```diff
@@ -76,6 +76,9 @@ cryptography = "!= 37.0.3"
 # version of tox to keep deterministic builds.
 tox = ">=4"
 
+# josepy 2.x introduces backwards incompatible changes
+josepy = "<2.0"
```

Comment:
I cannot imagine why this would be necessary? Am I missing something?

Reply:
i assumed we needed this to prevent poetry from somehow solving josepy 2.x as a pinned version

Reply:
It should do that based on the restrictions in the `setup.py`s. This file is for restrictions outside of that and weird edge cases.

Reply:
If I'm wrong and it allows 2.x despite the `setup.py` changes, I have misunderstood how this system works. Which is possible.

Reply:
i'd strongly bet on your understanding of the system over mine -- removing it!